api/service/testdata/model_endpoint_alert.yaml

```diff
@@ -27,7 +27,7 @@ groups:
             summary: Throughput (RPM) of model-1 model in env-1 is less than 2.00. Current value is {{ $value }}.
         - alert: "[merlin] model-1: 99.00p Latency warning"
           expr: |-
-            histogram_quantile(0.99, sum by(le, revision_name) (rate(revision_request_latencies_bucket{cluster_name="cluster-1",namespace_name="project-1",revision_name=~".*model-1.*"}[1m]))) > 3
+            histogram_quantile(0.99, sum by (le, revision_name) (rate(revision_request_latencies_bucket{cluster_name="cluster-1",namespace_name="project-1",revision_name=~".*model-1.*"}[1m]))) > 3
```

Comment:
These changes seem to have been caused by the new parser (or it might've just been due to some updated prometheus function that we're using) but it's pretty minor nonetheless.